src/javascripts/components/ChatInput.js

```diff
@@ -1,14 +1,30 @@
-var React = require('react');
+import React from 'react';
 
 class ChatInput extends React.Component {
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const { onSubmit } = this.props;
+    const text = this.inputText.value.trim();
+
+    if (this.inputText.value === "" || this.inputText.value === " ") {
```

Comment:
After the trim, I don't think you need the `this.inputText.value === " "` check.

Another way to do `this.inputText.value === ""`:
`if (this.inputText.value.length)` or `if (this.inputText.value.length > 0)`

The first one is valid since `0` is falsey